src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Literal.cs

```diff
@@ -18,6 +18,19 @@ public override BoundNode VisitLiteral(BoundLiteral node)
 
         private BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue constantValue, TypeSymbol type, BoundLiteral oldNodeOpt = null)
         {
+            return MakeLiteral(syntax, constantValue, type, _compilation, _factory.CurrentMethod, _inExpressionLambda, oldNodeOpt);
+        }
+
+        /// <summary>
+        /// Pass in 'currentMethod' and 'inExpressionLambda' if this is called during lowering, otherwise set 'currentMethod' to null (i.e. for IOperation purposes) 
```

Comment:
>'currentMethod' to null [](start = 110, length = 23)

It feels like this won't give us an accurate tree. Either this method should take currentMember as a Symbol and properly handle initializers, or, for the purposes of IOperation, we should create literals in their high level form as BoundLiteral nodes. My preference is on the latter.